README.md

```diff
@@ -177,6 +177,18 @@ combine them to produce and return a new stamp.
 
 Take an object and return true if it's a stamp, false otherwise.
 
+## API Differences
+
+react-stampit removes the following methods available in [stampit](https://github.com/stampit-org/stampit) core to enforce an API familiar to React:
+
+* init
+* props
+* refs
+* methods
+* statics
+
+Users are encouraged to utilize [React's lifecycle](https://facebook.github.io/react/docs/component-specs.html) and [component properties](#what-is-this) as replacements for these methods.
```

Comment:
react-stampit utitlizes a stamp description object made specifically for React components. Consider it a long lost relative of [stampit](https://github.com/stampit-org/stampit)'s stamp description object with nothing in common. react-stampit has also stripped all but the above mentioned static methods to enforce an API familiar to React users.